### PR TITLE
Add popper component and popperjs package

### PR DESCRIPTION
**Pull request by samuveth**

### CODE REVIEW

File: src/composables/useDebounce.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export function useDebounce() {
+  let timeout: any = null;
+  return function (fnc, delayMs) {
+    clearTimeout(timeout);
+    timeout = setTimeout(() => {
+      fnc();
+    }, delayMs || 500);
```
Comment (ChaituVR) [inline src/composables/useDebounce.ts:7]: Is it good to reduce this delay? 🤔 on mouse leave, it feels like it is taking some time for popper to go away.

**Comment by bonustrack** [comment]
Looks great! Can you make that the popover start exactly where the link start. It should start at the red line for voter for example:
image

**Comment by bonustrack** [comment]
Or at least make sure there is some minimal space on left, so popunder doesnt appear directly in the border like this:
image

File: src/composables/useDebounce.ts (hunk 1 above)
Comment (bonustrack) [inline src/composables/useDebounce.ts:7]:
```suggestion
    }, delayMs || 800);
```

**Comment by bonustrack** [comment]
On Twitter the popover appear in middle when there is space:
image
But aligned to left when there isn't space like on mobile:
image
Is this something possible here with popper?

**Comment by ChaituVR** [comment]
Is it possible to use modal (like previous) in smaller devices ..or maybe in smaller devices it should be always center of screen (like position: fixed) 

because it looks weird in iPhone SE
image

and not centered 
image

**Comment by ChaituVR** [comment]
and delay seems way more after clicking, is there a way to reduce that 🤔 
space new1

**Comment by samuveth** [comment]
> and delay seems way more after clicking, is there a way to reduce that 🤔

I changed it to open and close instantly, feel more intuitive as it requires a click on mobile.

**Comment by samuveth** [comment]
> On Twitter the popover appear in middle when there is space:

This would require us splitting up the User component into two if we want it center exactly at the user Icon. Currently the address and icon are together.

**Comment by bonustrack** [comment]
On small screen it go full width
image
Can it stay like the others screen size instead, like this:
image

**Comment by samuveth** [comment]
> Can it stay like the others screen size instead, like this:

@bonustrack each mobile screen width is a bit different and the popper will look bad on some if the width isn't full.

**Comment by bonustrack** [comment]
[Change user modal to popover](https://trello.com/c/0M1TxU7f/132-change-user-modal-to-popover)

**Comment by samuveth** [comment]
I'll do some more work on this after https://github.com/snapshot-labs/snapshot/pull/426 is merged.

**Comment by bonustrack** [comment]
We dont need to show the popover on small screen

**Comment by samuveth** [comment]
@bonustrack Now the popover will only open on mouse hover, if it's a touch screen it will not show on click. I also added all media queries from Primer and an extra one for XSmall with 420px.

**Comment by samuveth** [comment]
Also screen size detection is now reactive.

File: src/components/User.vue [hunk 2]
```diff
@@ -1,35 +1,62 @@
...
+          >
+            <UiButton class="button-outline width-full">
+              {{ $t('view3box') }}
+              <Icon name="external-link" class="ml-1" />
+            </UiButton>
+          </a>
+        </div>
+      </template>
+    </UiPopover>
+    <teleport to="#modal"> </teleport>
```
Comment (bonustrack) [inline src/components/User.vue:53]: This is not being used anymore isnt?

**Review by bonustrack — APPROVED** [review]
Works well now!

**Pull request merged**